alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -161,7 +164,7 @@ abstract class BaseAlertPresenter(private val alert: Alert) : AlertActions {
          * @param handler The block to execute after user taps a button
          */
         fun setPositiveButton(title: String, handler: AlertActionHandler = {}) = apply {
-            addAction(Alert.Action(title, Alert.Action.Style.POSITIVE, handler))
+            addAction(Alert.Action(title, Alert.Action.Style.POSITIVE, null, handler))
```

Comment:
you should add a secondary constructor that does not need this parameter to avoid breaking the API.

Reply:
6f1c51a